src/backend/main/py.main/caliopen_main/message/objects/message.py

```diff
@@ -112,6 +112,19 @@ def raw_json(self):
         msg = RawMessage.get_for_user(self.user_id, self.raw_msg_id)
         return json.loads(msg.json_rep)
 
+    @property
+    def external_msg_id(self):
+        return self.external_references.message_id if self.external_references \
+            else None
+
+    @property
+    def user_identity(self):
+        """
+        return first user_identity
+        """
+        return self.user_identities[0] if len(self.user_identities) > 0 \
```

Comment:
if self.user_identities is enough

Reply:
changed with commit #f55f2ace

Reply:
SeekMessageByExternalRef removed by commit #e1e3389e